cognite/neat/graph/stores/_base.py

```diff
@@ -221,7 +223,7 @@ def _parse_file(
         Args:
             filepath : File path to file containing graph data, by default None
             mime_type : MIME type of graph data, by default "application/rdf+xml"
```

Comment:
Revert to previous as base_uri is not always needed to load file to oxigraph. It is only needed when RDF file contains relative URIs, which is the case with Nordic44. In majority of cases there are absolute URI.

Reply:
Got it, will revert this change.